uploadNeuroDB/NeuroDB/ExitCodes.pm

```diff
@@ -100,6 +100,7 @@ our $MISSING_ARG          = 3; # if missing script's argument(s)
 our $DB_SETTINGS_FAILURE  = 4; # if DB settings in profile file are not set
 our $INVALID_PATH         = 5; # if path to file or folder does not exist
 our $INVALID_ARG          = 6; # if one of the program arguments is invalid
+our $INVALID_IMPORT       = 7; # if an import statement failed
```

Comment:
I like this addition, and it makes me think -- will we use this for MRI pipeline fails?  What is the difference between BIDS import fails and MRI pipeline fails?

Reply:
this lines add it to the ExitCodes.pm of the MRI pipeline (although it is not used for now but can be used in the future). There is another exitcode.py file for the BIDS import in Python. I want to keep both of them with the same exit codes, which is why I synced the file here.